src/components/MenuItem.tsx

```diff
@@ -178,6 +178,9 @@ type MenuItemBaseProps = {
     /** Text that appears above the title */
     label?: string;
 
+    /** Limit to truncate description for menu item */
```

Comment:
Is this limit in characters? 

```suggestion
    /** Character limit after which the menu item text will be truncated */
```